iotwifi/wpacfg.go

```diff
@@ -219,8 +253,8 @@ func (wpa *WpaCfg) ConnectNetwork(creds WpaCredentials) (WpaConnection, error) {
 }
 
 // Status returns the WPA wireless status.
-func (wpa *WpaCfg) Status() (map[string]string, error) {
-	cfgMap := make(map[string]string, 0)
+func (wpa *WpaCfg) Status() (map[string]interface{}, error) {
```

Comment:
Was this not able to be a string to string map anymore?

Reply:
This function can - another leftover mod from when I planned to embed the AP status in this endpoint.  Will fix